PWGJE/Tasks/hffragmentationfunction.cxx

```diff
@@ -123,7 +194,7 @@ struct HfFragmentationFunctionTask {
         double z_parallel = (jetVector * d0Vector) / (jetVector * jetVector);
 
         // calculating angular distance in eta-phi plane
-        axisDistance = sqrt(pow(jet.eta() - d0Candidate.eta(), 2) + pow(jet.phi() - d0Candidate.phi(), 2));
+        axisDistance = sqrt(pow(DeltaPhi(jet.eta(), d0Candidate.eta()), 2) + pow(jet.phi() - d0Candidate.phi(), 2));
```

Comment:
the deltaphi should be around phi not eta

Reply:
Don't know how I didn't notice that. Thanks for pointing out, will correct the other calculations as well